index.html

```diff
@@ -0,0 +1,850 @@
+            <h3>
+              The Complete Salesforce Trailhead 2020: From Zero to Hero 7-Course
+              Bundle
+            </h3>
+            <p>$299 $25</p>
+          </div>
+        </div>
+        <div class="grid1 sec2-media">
+          <img class="flex-img" src="images/offers.png" alt="pic here" />
+          <div class="flex-txt">
+            <h3>
+              The Complete Salesforce Trailhead 2020: From Zero to Hero 7-Course
+              Bundle
+            </h3>
+            <p>$299 $25</p>
+          </div>
+        </div>
+        <div class="grid1 sec2-media">
+          <img class="flex-img" src="images/offers.png" alt="pic here" />
+          <div class="flex-txt">
+            <h3>
+              The Complete Salesforce Trailhead 2020: From Zero to Hero 7-Course
+              Bundle
+            </h3>
+            <p>$299 $25</p>
+          </div>
+        </div>
+        <div class="grid1 sec2-media">
+          <img class="flex-img" src="images/offers.png" alt="pic here" />
+          <div class="flex-txt">
+            <h3>
+              The Complete Salesforce Trailhead 2020: From Zero to Hero 7-Course
+              Bundle
+            </h3>
+            <p>$299 $25</p>
+          </div>
+        </div>
+        <div class="grid1 sec2-media">
+          <img class="flex-img" src="images/offers.png" alt="pic here" />
+          <div class="flex-txt">
+            <h3>
+              The Complete Salesforce Trailhead 2020: From Zero to Hero 7-Course
+              Bundle
+            </h3>
+            <p>$299 $25</p>
+          </div>
+        </div>
+        <div class="grid1 sec2-media">
+          <img class="flex-img" src="images/offers.png" alt="pic here" />
+          <div class="flex-txt">
+            <h3>
+              The Complete Salesforce Trailhead 2020: From Zero to Hero 7-Course
+              Bundle
+            </h3>
+            <p>$299 $25</p>
+          </div>
+        </div>
+      </div>
+    </section>
+    <footer class="footer">
+      <div class="top-ftr flex">
+        <div class="ftr-icons flex">
+          <i class="fa fa-facebook-official" aria-hidden="true"></i>
+          <i class="fa fa-twitter" aria-hidden="true"></i>
+          <i class="fa fa-youtube-play" aria-hidden="true"></i>
+          <i class="fa fa-instagram" aria-hidden="true"></i>
+          <i class="fa fa-pinterest-square" aria-hidden="true"></i>
+          <i class="fa fa-rss" aria-hidden="true"></i>
+          <i class="fa fa-envelope" aria-hidden="true"></i>
+        </div>
+        <div class="ftr-links flex">
+          <a href="#">Events</a>
+          <a href="#">About</a>
+          <a href="#">Advertise</a>
+          <a href="#">Jobs</a>
+          <a href="#">contacts</a>
+        </div>
+      </div>
+      <div class="lower-ftr flex">
+        <p>
+          © 2006–2020 The Next Web B.V. Made with 3 in Amsterdam. Powered by
+          StackPath
+        </p>
+      </div>
+    </footer>
```

Comment:
#### All screen sizes

- [ ] The icons are broken now.

![image](https://user-images.githubusercontent.com/61479648/91843389-11d35480-ec2c-11ea-8024-a759a7d80ba3.png)

#### Mobile Screen Size

- [ ] The bottom part should have smaller fonts

| Yours | Original |
|:---:|:---:|
| ![image](https://user-images.githubusercontent.com/61479648/91843493-33344080-ec2c-11ea-8992-434892a4c2c5.png)| ![image](https://user-images.githubusercontent.com/61479648/91843520-40e9c600-ec2c-11ea-9b07-dcf236cfc6b2.png) |